source/v4/teams.yaml

```diff
@@ -40,6 +40,36 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+    '/teams':
+    get:
+      tags:
+        - teams
+      summary: Get a list of all teams.
+      description: |
+        Logged in, only shows open teams. Shows all teams with "manage_system" permission. The result is based on query string parameters - page and per_page.
```

Comment:
Can you change this to `For regular users only returns open teams. Users with the "manage_system" permission will return teams regardless of type. The result is based on query string parameters - page and per_page.`